.gitignore

```diff
@@ -19,3 +19,6 @@ relic/
 *wpu
 .pytest_cache
 results.xml
+*json
```

Comment:
Is there a reason these are added to `.gitignore`?  I don't think we want to to do this, as actually do have JSON and FITS files that are currently part of the repo.

If you're trying to make sure temporary outputs of tests don't get into the repo, then use the pytest fixture `tmpdir` with the test.